mullvad-management-interface/proto/management_interface.proto

```diff
@@ -235,6 +238,24 @@ message TunnelEndpoint {
   bool daita = 9;
 }
 
+message FeatureIndicators { repeated FeatureIndicator active_features = 1; }
+
+enum FeatureIndicator {
+  QUANTUM_RESISTANCE = 0;
+  MULTIHOP = 1;
+  BRIDGE_MODE = 2;
+  SPLIT_TUNNELING = 3;
+  LOCKDOWN_MODE = 4;
+  UDP_2_TCP = 5;
+  LAN_SHARING = 6;
+  DNS_CONTENT_BLOCKERS = 7;
+  CUSTOM_DNS = 8;
+  SERVER_IP_OVERRIDE = 9;
+  CUSTOM_MTU = 10;
+  CUSTOM_MSS_FIX = 11;
+  DAITA = 12;
+}
```

Comment:
Just a thought, but `FeatureIndicators` and `FeatureIndicator` have very similar names. What do you think about replacing `FeatureIndicators` with e.g. `FeatureIndicatorSet` or something similar?

Reply:
I don't really mind the close resemblance, but I see how it could be confusing. I like the relative brevity of it though.  I'll think about it :blush: